utilities/blob_db/blob_db_test.cc

```diff
@@ -688,11 +689,31 @@ TEST_F(BlobDBTest, DISABLED_GCOldestSimpleBlobFileWhenOutOfSpace) {
 
 TEST_F(BlobDBTest, ReadWhileGC) {
   // run the same test for Get(), MultiGet() and Iterator each.
+  std::shared_ptr<SstFileManager> sst_file_manager(
+      NewSstFileManager(mock_env_.get()));
+  sst_file_manager->SetDeleteRateBytesPerSecond(1);
```

Comment:
You might want to pull this new functionality into a new test, specifically testing blob files going through the delete scheduler. "ReadWhileGC" does not accurately reflect that, imo.